README.md

```diff
@@ -34,8 +97,15 @@ you can have a copy of this project by cloning this repo to your local machine
 
 with the steps above, you can have a copy of the project 
 
+
 ## Author
 
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
 👤 **Author**
 
 - GitHub: [@githubhandle](https://github.com/Ademola-coding)
```

Comment:
Kindly update your README file as per [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template.  You can customize your README file however you want, but the main headers from this provided template README file should be present. Also, kindly remove the sections which are not relevant to your project like Getting Started with Create React App, Available scripts, etc.

Reply:
THANKS